src/stepper/__tests__/stepper.ts

```diff
@@ -9,6 +9,10 @@ function getLastStepAsString(steps: [substituterNodes, string[][], string][]): s
   return codify(steps[steps.length - 1][0]).trim()
 }
 
+function getExplanation(steps: [substituterNodes, string[][], string][]): string {
+  return steps.map(x => x[2]).join('\n')
```

Comment:
What is the `[2]` here? Can we add a comment?

Reply:
Left a comment for now, in the future all the indexes of steps should be accessed through a constant. Or maybe steps should just be an object array instead for better clarity...